### DISCUSSION TITLE
Do I need to remove the site of strong Linkage disequilibrium(LD)

### DISCUSSION THREAD

**Discussion by Tom159357** [vegandevs/vegan#580] [Q&A]
When analyzing the relationship between genotype and environmental factors, do I need to run Linkage disequilibrium prune before redundancy analysis?

**Comment by gavinsimpson** [comment]
You're unlikely to get many informed eyes on your problem here; try asking at a more relevant venue (somewhere that the analysis of this kind of data is more common).

The obvious answer is no; RDA doesn't *require* this pruning. So your question is more as to the need for such pruning in relation to your scientific question, which as yet hasn't been stated.
